plugin-maven/src/test/resources/pom-test.xml.mustache

```diff
@@ -21,6 +21,7 @@
 
     <build>
         <plugins>
+            {{{plugins}}}
```

Comment:
I see this was added back in and cascaded through `MavenIntegrationHarnesss` but it looks like it's not used anywhere. Is this an artifact of fixing a merge conflict with main? Or maybe I'm missing a place where it is used...

Reply:
I had to check myself again, but found the spot where it is used:

`NpmTestsWithDynamicallyInstalledNpmInstallationTest.useDownloadedNpmInstallation`:

https://github.com/simschla/spotless/blob/e8f0d2d27bb542987b8cc97b7a4c2b5a04880306/plugin-maven/src/test/java/com/diffplug/spotless/maven/npm/NpmTestsWithDynamicallyInstalledNpmInstallationTest.java#L29-L35

There I add the plugin `frontend-maven-plugin` to dynamically download and extract npm/node for a maven build.